modules/deploy/azure/output.tf

```diff
@@ -5,7 +5,7 @@
 #
 
 output "public_ip" {
-  value = azurerm_linux_virtual_machine.cml.public_ip_address
+  value = azurerm_public_ip.cml.ip_address
```

Comment:
same here = "workaround for 10m timeout error ( ignore_errors=true does not work and we also do not need this), since we only want public ipv4 under which will be the cml2 accessible"